packages/elements/src/components/ino-dialog/ino-dialog.tsx

```diff
@@ -125,7 +159,27 @@ export class Dialog implements ComponentInterface {
               aria-modal="true"
             >
               <div tabindex="0" />
-              <slot />
+              {!hasContentSlot?
+                <div> 
+                  {this.headline? 
+                    <div class="headline">
+                      {this.icon && <ino-icon icon={this.icon} />}
+                      <h1>{this.headline}</h1>
+                    </div>
+                  : null}
+                  {
+                    this.description?
+                      <div class="description">
+                        {this.description}
+                      </div> 
+                  : null}
+                  <slot name="body"/>
+                  <div class={`${'button-row'} ${!(this.description) && 'big-margin'}`}>
+                    {this.cancelbutton? <ino-button  variant="outlined">{this.cancelbutton}</ino-button> : null}
```

Comment:
Instead of using the ternary operator (`condition ? true : false`) you can just use the `&&`. In this case it would be:

```suggestion
                    {this.cancelbutton && <ino-button  variant="outlined">{this.cancelbutton}</ino-button>}
```